scripts/format-ros2.sh

```
@@ -255,20 +255,15 @@ check_package_cpplint() {
     local package_name="${rel_path%%/*}"
-    local filename
-    filename=$(basename "$file" .hpp)
-    
-    local package_upper
-    package_upper=$(echo "$package_name" | tr '[:lower:]-' '[:upper:]_')
-    local filename_upper
-    filename_upper=$(echo "$filename" | tr '[:lower:]-' '[:upper:]_')
-    
-    echo "${package_upper}__${filename_upper}_HPP_"
+
+    local package=$(echo "$package_name" | tr '[:lower:]-' '[:upper:]_')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)